lib/turn.rb

```diff
@@ -48,7 +47,13 @@ def player_attacks(attack_coords = get_coords)
 
   def get_coords
     p "What coordinates would you like to attack? For example, you can enter A1:"
-    gets.chomp.upcase.gsub(/\s+/, "")
+    user_input = gets.chomp
+    user_coords = user_input.upcase.gsub(/\s+/, "")
+    if user_input.to_s.strip.upcase == "EXIT"
```

Comment:
I dunno if putting this if statement anytime there is user input is the best way? Can they quite in the middle of gameplay?